### PR TITLE
link to configuring plugin page from all plugin page

### LINKED ISSUE #1582: reference `configure_plugin` page in all plugins doc
a follow up to https://github.com/facebookresearch/hydra/pull/1506

### PR DESCRIPTION
**Pull request by jieru-hu**
closed #1582

### CODE REVIEW

File: website/docs/plugins/optuna_sweeper.md [hunk 1]
```diff
@@ -40,6 +40,8 @@ defaults:
 
 Alternatively, add `hydra/sweeper=optuna` option to your command line.
 
+
+
 The default configuration is <GithubLink to="plugins/hydra_optuna_sweeper/hydra_plugins/hydra_optuna_sweeper/config.py">here</GithubLink>.
```
Comment (Jasha10) [inline website/docs/plugins/optuna_sweeper.md:45]: Looks like only whitespace was added to `optuna_sweeper.md`

File: website/docs/plugins/ax_sweeper.md [hunk 2]
```diff
@@ -115,3 +115,4 @@ ax_config:
   is_noisy: true
   params: {}
 ```
+You have multiple options configuring the Plugin, check out [this page](../patterns/configuring_plugins) for more info.
```
Comment (omry) [inline website/docs/plugins/ax_sweeper.md:118]:
```suggestion
There are several standard approaches for configuring plugins. Check [this page](../patterns/configuring_plugins) for more info.
```

File: website/docs/plugins/ax_sweeper.md [hunk 3]
```diff
@@ -115,4 +115,4 @@ ax_config:
   is_noisy: true
   params: {}
 ```
-You have multiple options configuring the Plugin, check out [this page](../patterns/configuring_plugins) for more info.
+There are several standard approaches for configuring plugins. Check [this page](../patterns/configuring_plugins) for more info.
```
Comment (omry) [inline website/docs/plugins/ax_sweeper.md:118]:
```suggestion
There are several standard approaches for configuring plugins. Check [this page](../patterns/configuring_plugins) for more information.
```

Sorry :)

Comment (jieru-hu) [reply website/docs/plugins/ax_sweeper.md]: 😆 fixed!

**Review by omry — APPROVED** [review]

**Pull request merged**